Libraries/vendor/tinycolor/tinycolor.js

```diff
@@ -1,503 +1,341 @@
 /**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
  * @providesModule tinycolor
- * @nolint
+ * @flow
  */
+/* eslint no-bitwise: 0 */
 'use strict';
 
-// TinyColor v1.2.1
```

Comment:
At this point I reimplemented the entire file and there's no line that has been unchanged. We can drop the header